core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.scss

```diff
@@ -30,11 +30,11 @@
             overflow-y: hidden;
         }
 
-        .dot-template-builder__new-template-builder {
-            .p-tabview-nav {
-                padding: 0 $spacing-5;
-            }
-        }
+        // .dot-template-builder__new-template-builder {
```

Comment:
If this code is not used, can it be deleted?

Reply:
Im testing. I will uncomment it